app/src/main/java/com/duckduckgo/app/settings/SettingsViewModel.kt

```diff
@@ -69,7 +75,8 @@ class SettingsViewModel @Inject constructor(private val settingsDataStore: Setti
         Timber.i("User changed autocomplete setting, is now enabled: $checked")
         settingsDataStore.autoCompleteSuggestionsEnabled = checked
     }
-    private fun obtainVersion(): String {
-        return "${BuildConfig.VERSION_NAME} (${BuildConfig.VERSION_CODE})"
+    private fun obtainVersion(variantKey: String): String {
+        val version = "${BuildConfig.VERSION_NAME} (${BuildConfig.VERSION_CODE})"
+        return "$version variant=($variantKey)"
```

Comment:
The `variant=()` text looks too techie to me. How would you feel about listing it next to the version number e.g `5.7.5 ma (50705)`?

Reply:
Good point; i've updated to use your suggestion